Nightly Bridge Integration: Rust Format Error & Node.js 20 Deprecation Warning

### Description
The [Nightly Bridge Integration](https://github.com/d-oit/do-web-doc-resolver/actions/runs/24945350612/job/73045757574#step:10:17) workflow failed with the following issues:

1. **Rust Format Error**
   - Step: `Format Rust`
   - Error: Process completed with exit code 1.

2. **Node.js 20 Deprecation Warning**
   - The workflow uses `actions/cache@v4`, which is running on Node.js 20. This version is deprecated and will be removed from the runner on September 16th, 2026.
   - Starting June 2nd, 2026, actions will be forced to run with Node.js 24 by default.
   - **Action Required:** Update to a version of `actions/cache` that supports Node.js 24, or set the `FORCE_JAVASCRIPT_ACTIONS_TO_NODE24=true` environment variable to opt in early.

### Steps to Reproduce
- Trigger the Nightly Bridge Integration workflow.
- Observe the failure in the `Format Rust` step and the Node.js 20 deprecation warning.

### Expected Behavior
- The workflow should complete successfully without errors or warnings.
- All actions should be compatible with Node.js 24.

### Additional Context
- Workflow run: [#24945350612](https://github.com/d-oit/do-web-doc-resolver/actions/runs/24945350612)
- Job: [#73045757574](https://github.com/d-oit/do-web-doc-resolver/actions/runs/24945350612/job/73045757574)

### Proposed Solution
1. Investigate the Rust format error and fix the underlying issue.
2. Update the workflow to use a version of `actions/cache` compatible with Node.js 24, or set the required environment variable to opt in early.

/label bug
/label workflow
/label rust
/label nodejs

--- a/.github/workflows/nightly-bridge.yml
+++ b/.github/workflows/nightly-bridge.yml
@@ -26,9 +26,10 @@ jobs:
         uses: dtolnay/rust-toolchain@stable
         with:
           toolchain: '1.85'
+          components: rustfmt
 
       - name: Cache Rust dependencies
-        uses: actions/cache@v4
+        uses: actions/cache@v5
         with:
           path: |
             ~/.cargo/bin/

